html/event.html

```diff
@@ -41,11 +42,11 @@ <h1 class="text-white text-2xl font-semibold">NTC</h1>
             </label>
             <ul class="hidden md:flex md:space-x-4 text-white">
                 <li><a href="../index.html">Home</a></li>
-                <li><a href="#introduction">Introduction</a></li>
-                <li><a href="#events">Events</a></li>
-                <li><a href="#team">Executive Team</a></li>
-                <li><a href="#gallery">Gallery</a></li>
-                <li><a href="#contact">Contact Us</a></li>
+                <li><a href="../index.html">Introduction</a></li>
+                <li><a href="../index.html">Events</a></li>
+                <li><a href="../index.html">Executive Team</a></li>
+                <li><a href="../index.html">Gallery</a></li> <!-- Updated link to point to index.html's gallery section -->
+                <li><a href="../index.html">Contact Us</a></li>
```

Comment:
Can you please, rollback these changes back to the page sections.

@prakash27x

Reply:
Ohh sorry got it. I was thinking its index page.